corehq/apps/commtrack/static/commtrack/ko/products_and_programs.async.js

```diff
@@ -73,7 +73,7 @@ var CommtrackProductsProgramsViewModel = function (o) {
 $.fn.asyncProgramProductList = function (options) {
     this.each(function(i, v) {
         var viewModel = new CommtrackProductsProgramsViewModel(options);
-        ko.applyBindings(viewModel, $(this).get(i));
+        $(this).koApplyBindings(viewModel);
```

Comment:
Yeah I think this is the first one, but if you search for `get(i)` and `get(j)` (maybe you have a better way to search with regex or something) you'll see a bunch here. Some of them seem to have been carefully thought about by replacing the looping with something else, but I think most of them are an oversight?